modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -304,7 +304,7 @@ else {
 	}
 </aui:script>
 
-<aui:script require="users-admin-web/js/actions.es as actions">
+<aui:script require="<%= npmResolvedPackageName + "/js/actions.es as actions"%>">
```

Comment:
For some reason I was under the impression that if the js file you are referencing is in the same module, you only need to lead with a forward slash to reference resources. Similar to how `frontend:component` takes something like `/js/MyComponent.js`

Reply:
Yeah, I thought that too, but I checked the other day and it's not the case for `<aui:script>`. We can do it automagically in other tags because:

1. They live in OSGi land (`<aui:script>` doesn't) so they have access to npm support classes
2. They are not legacy, so we can add new things to them

In the case of `<aui:script>` we cannot access any OSGi thing because it is outside of OSGi and also, we don't want to implement more than it's needed to be able to deprecate it some day. If we allow referencing any npm module (other than the internal js modules) we are reducing the friction to keep JS code inside JSPs, which is a bad thing.